earth_enterprise/src/server/geresetpgdb

```diff
@@ -138,39 +162,54 @@ elif [ $type == "soft" ]; then
       exit
       ;;
   esac
-elif [ $type == "restore" ]; then
-  if [ ! -n "$2" ]; then
+elif [ $type == "restore" ] || [ $type == "upgrade" ] || [ $type == "backup" ]; then
+  if [ $type == "restore" ] && [ ! -n "$2" ]; then
     echo "WARNING: Path to dump directory is not specified."
     echo "Usage: $0 restore path_to_dump_dir"
     exit
   else
-    dump_path="$2"
+    if [ -n "$2" ]; then
+      dump_path="$2"
+    fi
   fi
 fi
 
 # Dumps geserve-databases.
 function do_dump {
   # Start postmaster.
+  echo Do data dump!
```

Comment:
Was this just for debugging or is it intended to be permanent?

Reply:
It could be permanent, but I should probably make it more useful, like where is it dumping data to